cime_config/buildnml

```diff
@@ -208,7 +208,10 @@ def buildnml(case, caseroot, compname):
 
             # marbl_in and marbl_diagnostics_operators
             if marbl_nt_build > 0:
-                _copy_file('marbl_in', confdir, rundir, inst_string)
```

Comment:
I think the `pop-ecosys_multiinst_lsource_sink` test is in `aux_pop_MARBL` but not `aux_POP` or any other testlist. Could you please verify that `SMS_N2.[resolution].C1850ECO.[mach_compiler].pop-ecosys_multiinst_lsource_sink` is bit-for-bit with the old version? It seems like this change might change the behavior of that test?

Reply:
@mnlevy1981 SMS_N2.T62_g17.C1850ECO baseline test passes (compared with 4bb66e1c89e8dabd55cbd416d8e831c37ceb3262 )

I can go ahead and run full aux_pop and aux_pop_MARBL if there aren't any other issues.

Reply:
@alperaltuntas Looks good to me!